android/app/build.gradle

```diff
@@ -9,7 +9,6 @@ buildscript {
 
 apply plugin: 'com.onesignal.androidsdk.onesignal-gradle-plugin'
 apply plugin: "com.android.application"
-apply from: '../../node_modules/react-native-unimodules/gradle.groovy'
```

Comment:
We had a line:

```
apply from: new File(["node", "--print", "require.resolve('react-native/package.json')"].execute().text.trim(), "../react.gradle")
```
in the hotspot app. I saw that in the migration guild, was it not needed?

Reply:
hmmm. I just missed this, but it worked fine without 🤷‍♂️